web-app/django/VIM/apps/instruments/views/instrument_list.py

```diff
@@ -1,7 +1,14 @@
+from typing import Any
 from django.views.generic import ListView
 from VIM.apps.instruments.models import Instrument
 
 class InstrumentList(ListView):
     template_name = "instrument_list.html"
     context_object_name = "instruments"
     model = Instrument
+
+    def get_paginate_by(self, queryset):
+        paginate_by = self.request.GET.get("paginate_by", None)
```

Comment:
`paginate_by: str = self.request.GET.get("paginate_by", "20")`?

But also, a type annotation on this would show you that you are returning a `str` if you return `paginate_by`, and an `int` if you don't have a `paginate_by` query argument. You also need to be prepared that someone would pass `?paginate_by=foo" in which case simply casting to an `int` would raise an exception.